src/app/common/templates/config-data.yaml

```diff
@@ -33,6 +33,9 @@ CommandHandlerInterfaceOnlyClusters:
     - ActivatedCarbonFilterMonitoring
     - HepaFilterMonitoring
     - RvcOperationalState
+    # cluster format should match ClustersWithInitFunctions et al
+    # See https://github.com/project-chip/zap/issues/1134
```

Comment:
That needs a Matter issue, not a ZAP issue....  This is not handled ZAP-side at all.

Reply:
https://github.com/project-chip/connectedhomeip/issues/29186